pkg/cloudconformity/create_conformity_custom_rule.go

```diff
@@ -19,7 +19,7 @@ func (c *Client) CreateConformityCustomRule(payload CustomRuleRequest) (*CustomR
 
 	log.Printf("[DEBUG] Conformity CreateCustomRule request payload: %v\n", string(rb))
 
-	_, err = c.ClientRequest(Post{}, "/custom-rules/", strings.NewReader(string(rb)), "", &response)
+	_, err = c.ClientRequest(Post{}, "/custom-rules", strings.NewReader(string(rb)), "", &response)
```

Comment:
⛏️ Might as well put this back since we didn't need to change it
```suggestion
	_, err = c.ClientRequest(Post{}, "/custom-rules/", strings.NewReader(string(rb)), "", &response)
```